src/arrayMethodJoin.js

```diff
@@ -4,8 +4,19 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    const strSeparator = String(separator);
```

Comment:
Why do you need to create this constant? You'll always add this separator to string and it will be converted to sting by default

'' + null === 'null'
'' + undefined === 'undefined'